sway-lib-std/src/outputs.sw

```diff
@@ -61,7 +61,7 @@ pub fn output_pointer(index: u64) -> u64 {
 
 /// Get the transaction outputs count for either tx type
 /// (transaction-script or transaction-create).
-pub fn output_count() -> u64 {
+pub fn output_count() -> u8 {
```

Comment:
This means that this is breaking change right?

Reply:
I reverted this; not sure why I changed it.